.github/workflows/ci_file_health.yaml

```diff
@@ -34,11 +36,23 @@ jobs:
             -o "${{ runner.temp }}/pandoc.tar.gz" \
             "https://github.com/jgm/pandoc/releases/download/3.6/pandoc-3.6-linux-amd64.tar.gz"
           tar -xf "${{ runner.temp }}/pandoc.tar.gz" -C "${{ github.workspace }}/.."
-          mv "${{ github.workspace }}/.."/pandoc-* "${{ env.pandoc_path }}"
+          mv "${{ github.workspace }}/.."/pandoc-* "$pandoc_path"
           # run pandoc
           for lang in doc/*/; do
-            "${{ env.pandoc_path }}/bin/pandoc" -f markdown -t man -s "$lang/qbittorrent.1.md" -o "$lang/qbittorrent.1"
-            "${{ env.pandoc_path }}/bin/pandoc" -f markdown -t man -s "$lang/qbittorrent-nox.1.md" -o "$lang/qbittorrent-nox.1"
+            "$pandoc_path/bin/pandoc" -f markdown -t man -s "$lang/qbittorrent.1.md" -o "$lang/qbittorrent.1"
+            "$pandoc_path/bin/pandoc" -f markdown -t man -s "$lang/qbittorrent-nox.1.md" -o "$lang/qbittorrent-nox.1"
           done
           # check diff, ignore "Automatically generated by ..." part
           git diff -I '\.\\".*' --exit-code
+
+      - name: zizmor workflow audits
+        run: |
```

Comment:
It seems it can also be integrated via `pre-commit`:
https://woodruffw.github.io/zizmor/usage/#use-with-pre-commit

Reply:
> like this? https://github.com/qbittorrent/qBittorrent/pull/22120/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9

👍

Reply:
>Does that mean I should remove the workflow?

Yes. To be exact, revert changes to file_health workflow. I'll handle the errors from zizmor in another PR if you don't mind.